src/cr/cube/cube.py

```diff
@@ -233,6 +231,11 @@ def counts(self):
     def counts_with_missings(self):
         return self._measure(self.is_weighted).raw_cube_array
 
+    @lazyproperty
+    def cube_index(self):
```

Comment:
out of curiosity: does this property returns a cube idx or an offset?

Reply:
`idx` and "offset" are generally synonymous. In this case, it is always a positive int value, the first cube returning `0`, the next `1`, etc. It is never `None` however.

Reply:
👍